scenarios/get_wf_runnning_status/README.MD

```diff
@@ -0,0 +1,35 @@
+# Workflow: Get running status of another workflow
+
+### Caution: This example is NOT recommended.
+
+## What is the purpose of this scenario?
+To get running status of another workflow
+
+## Regarding api_host
+"api_host" in get_wf_running_status.dig is up to your account's site.  
+You have to change it if your account's site is located in EU or Tokyo.
+
+Please refer to the doc for more details.  
+[https://support.treasuredata.com/hc/en-us/articles/360001474288-Sites-and-Endpoints](https://support.treasuredata.com/hc/en-us/articles/360001474288-Sites-and-Endpoints)
```

Comment:
[nitpick] In GitHub markdown, when using the same strings as link, you'd be able to write URL without `()[]`
https://guides.github.com/features/mastering-markdown/

```suggestion
https://support.treasuredata.com/hc/en-us/articles/360001474288-Sites-and-Endpoints
```